cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
```

Comment:
The 'viewportHeight' and 'viewportWidth' seem to be swapped. Typically, width is larger than height for screen dimensions. Please verify that these values are intended and represent the viewport size accurately.